src/detectors/HCAL/RawCalorimeterHit_factory_HcalEndcapPRawHits.h

```diff
@@ -39,10 +39,10 @@ class RawCalorimeterHit_factory_HcalEndcapPRawHits : public JFactoryT<edm4hep::R
         m_input_tag = "HcalEndcapPHits";
         u_eRes = {};
         m_tRes = 0.0 * dd4hep::ns;
-        m_capADC = 1024;
-        m_dyRangeADC = 3.6 * dd4hep::GeV;
-        m_pedMeanADC = 20;
-        m_pedSigmaADC = 0.8;
+        m_capADC = 4096;
+        m_dyRangeADC = 200. ;
```

Comment:
Please ensure units are used here

Reply:
Hi - I had to remove the units since dyRangeADC secretly adds them in CalorimeterHitReco.cc in L27 like this: 

`dyRangeADC = m_dyRangeADC * dd4hep::MeV;`

So If I presented it with MeV, then it would be scaled twice (which completely threw me off).  I would prefer dyRangeADC to be assumed to have energy units, and remove this hidden rescaling (unless I missed something).

Reply:
Hidden conversions are bad. Let me look into that. The problem is that we have issue with units. @wdconinc started to fix those and filter out what happens in calorimetry algorithms with units here:  https://github.com/eic/EICrecon/pull/365. But it has not yet been finished. And I started to put the correct default values from python to C++. With all that we decided that the correct implicit units shold be DD4Hep ones. @wdconinc is it known and fixed in your PR?

Reply:
#365 does indeed remove the explicit addition of MeV in CalorimeterHitReco.cc, https://github.com/eic/EICrecon/pull/365/files#diff-7a64db59fabe7d6949bed432036c5f5250a349e7e4d50fc7d4d20a39a96c6b36L27.